src/components/ModalCard.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  Typography, withStyles, Modal, Slide, Card, CardMedia, Avatar, CardContent,
+} from "@material-ui/core";
+import { bigStyles } from "screens/DashboardStyles";
+
+function ModalCard(props) {
+  const {
+    classes, modalData, openModal, onClose,
+  } = props;
+  const { birth_date, phone_number } = modalData[1];
```

Comment:
what is [1] seem like a magic number here

Reply:
It's
```
const userData = [
  props.value, // Full name
  props.original, // other data
 ];
```
which is set to state's `selectedUser`:
```
{
  selectedUser: [
    "",
    {
      birth_date: "",
      phone_number: "",
    },
  ],
  openModal: false,
}
```
Then I pass `this.state.selectedUser` into `<ModalCard/>`'s `modalData` prop.

Reply:
why is selectedUser have to be an array ?

Reply:
can you not call this modalcard, so generic, if this is profile, name it ProfileCard